app/static/styles/main.css

```diff
@@ -52,29 +52,28 @@ body {
 .profile {
     background-color: #1d539f !important;
     text-align: center;
-    height: 320px;
+    height: auto;
     margin-bottom: 50px;
 }
 
 .profile h1 {
-        font-size: 50px;
```

Comment:
Just curious, since it seems to have no bearing on the functionality of the code--why the change in whitespace?

Reply:
Oh, I'm using Prettier and it changed the indentation automatically

Reply:
I suggest that you all setup a single kind of environment with such extensions in your IDE so that the indentation and other changes are consistent